terraform/vars/terraform-production.tfvars

```diff
@@ -5,6 +5,8 @@ rw_api_url                = "https://api.resourcewatch.org"
 desired_count             = 2
 auto_scaling_min_capacity = 2
 auto_scaling_max_capacity = 15
+fargate_cpu               = 2048
+fargate_memory            = 4096
```

Comment:
Here I think you are undoing a change that Solomon explicitly made (had removed these lines).  So I would be careful with this backmerge from master to develop - make sure you're not undoing any changes in develop.